src/orange/orangeinp/ObjectIO.json.cc

```diff
@@ -197,6 +197,14 @@ void to_json(nlohmann::json& j, EllipticalCone const& cr)
          SIO_ATTR_PAIR(cr, upper_radii),
          SIO_ATTR_PAIR(cr, halfheight)};
 }
+void to_json(nlohmann::json& j, ExtrudedPolygon const& cr)
+{
+    j = {
+        {"_type", "extrudedpolygon"}, SIO_ATTR_PAIR(cr, polygon),
+        // SIO_ATTR_PAIR(cr, line_segment),
+        // SIO_ATTR_PAIR(cr, scaling_factors)
```

Comment:
Are these supposed to be commendted out?

Reply:
This is stale; it was resolved in a subsequent commit.